mdal/mdal_utils.hpp

```diff
@@ -139,6 +139,12 @@ namespace MDAL
     const std::string &s,
     const std::string &delimiters = " \f\n\r\t\v" );
 
+  std::string systemFileName( const std::string &utf8FileName );
+
+//#if defined(WIN32)
```

Comment:
`#if defined(WIN32)` or even better make it static function in mdal_utils.cpp and not add it to header at all.